packages/Webkul/Core/src/Models/Channel.php

```diff
@@ -93,7 +93,7 @@ public function logo_url()
             return;
         }
 
-        return Storage::url($this->logo);
+        return Storage::url('app/public/' . $this->logo);
```

Comment:
You can reproduce this problem for yourself. If you keep your codem then you go to `{domain}/public/admin/channels/edit/{channel_id}` and upload a logo photo. The URL generated (the path in DB `channels` table `logo` field) won't be correct, and the logo won't re-display.